distributions/atalanta-rtic/pcs-pass/Cargo.toml

```diff
@@ -13,4 +13,6 @@ syn = { version = "2.0.53", features = [
     "full",
 ] }
 rtic-core = { path = "../../../rtic-core" }
-atalanta-macros = { package = "atalanta-bsp-macros", path = "../../Atalanta/examples/atalanta-bsp/macros" }
+# atalanta-macros = { package = "atalanta-bsp-macros", path = "../../Atalanta/examples/atalanta-bsp/macros" }
+atalanta-macros = { package = "atalanta-bsp-macros", git = "https://github.com/soc-hub-fi/atalanta.git", tag = "v0.2.1" }
```

Comment:
It seems a local path slipped in. Please remove the commented out part. The git version is correct.